src/services/NotificationService.ts

```diff
@@ -28,6 +28,7 @@ class NotificationService {
     const emailDetails = {
       personalisation: params,
     };
+    // @ts-ignore
```

Comment:
why's this needed now when it wasn't before?

Reply:
The complier was happy with it not being declared when it wasn't being used later, now it is, it's moody. Not sure what is wrong with the types in all honesty, but this same technique is used in this lambda. Likely will be fixed by type control that comes later.